Files/EPAlertView.swift

```diff
@@ -33,7 +33,9 @@ open class EPAlertView: UIView {
   
   fileprivate func addNameField() {
     nameField.autocapitalizationType = .words
-    nameField.placeholder = EPLocalizationUtil.with(" Name")
+    nameField.placeholder = EPLocalizationUtil.with("Name")
```

Comment:
I think the right way to do it is with inset -- 
view.bounds = CGRectInset(view.frame, 10.0f, 10.0f);